hbase-common/src/main/java/org/apache/hadoop/hbase/util/UnsafeAccess.java

```diff
@@ -330,6 +330,20 @@ public static void copy(byte[] src, int srcOffset, ByteBuffer dest, int destOffs
     unsafeCopy(src, srcAddress, destBase, destAddress, length);
   }
 
+  /**
+   * Copies the bytes from given array's offset to length part into the given array.
+   * @param src source array
+   * @param srcOffset offset into source buffer
+   * @param dest destination array
+   * @param destOffset offset into destination buffer
+   * @param length length of data to copy
+   */
+  public static void copy(byte[] src, int srcOffset, byte[] dest, int destOffset, int length) {
```

Comment:
For copying from byte array to byte array, we could just use System.arraycopy? It is a native method.

Reply:
The unsafe version is faster than arraycopy because it does not do bounds checking, right? I don't have a strong opinion about this, though.

Reply:
I'll remove this, turns out it was for an earlier version of the patch and is no longer used.